### DISCUSSION TITLE
How I can get road distance without draw

### DISCUSSION THREAD

**Discussion by tamharo** [liodali/osm_flutter#344] [Q&A]
Hey, 
I need get road distance and duration without draw, like: 

RoadInfo = await controller.road(
 GeoPoint(latitude: 47.35387, longitude: 8.43609),
   GeoPoint(latitude: 47.4371, longitude: 8.6136),
   roadType: RoadType.car,
);
print("${roadInfo.distance}km");
print("${roadInfo.duration}sec");

distance2point return does not take the road just lat long calcul

Thank you in advance

**Comment by liodali** [comment]
technically we always need road to know the distance

**Reply by tamharo** [reply]
Technically yes, sorry, i meant the road by car, distance2point take road with all path, but car don’t take all

**Reply by liodali** [reply]
you neam you want feature to get distance without need of drawing 
take look at this package : https://github.com/liodali/OSM-Routing-Client-Dart
